stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -79,6 +79,66 @@ var result = websub:startHub(new http:Listener(8080), hubConfiguration = hubConf
 
 Any subscriptions added at the hub will now be available even when the hub is restarted.
 
+##### Enabling Basic Auth support for the hub
```

Comment:
Shall we make this the first section and make "Enabling data persistence for the hub" the second?